packages/create-atomic/plopfile.js

```diff
@@ -0,0 +1,58 @@
+const {spawn} = require('child_process');
+
+module.exports = function (
+  /** @type {import('plop').NodePlopAPI} */
+  plop
+) {
+  const currentPath = process.cwd();
+  plop.setGenerator('@coveo/atomic', {
+    description: 'A Coveo Atomic Generator',
+    prompts: [
+      {
+        type: 'input',
+        name: 'project',
+        message: 'Name of the project',
+        validate: (input) => {
+          const pattern =
+            '^(?:@[a-z0-9-*~][a-z0-9-*._~]*/)?[a-z0-9-~][a-z0-9-._~]*';
```

Comment:
voodoo regexp, what's that 😂

Reply:
That's NPM's official regex, I'll remove it and perhaps just kebab case the input

Reply:
It's OK to keep it, just 'document it', i.e. assign it to a variable and name it well, and add a link to your source.